docs/flutter-upgrade.md

```diff
@@ -0,0 +1,14 @@
+### Upgrade flutter
+
+####  Note: We are using fvm
+- check for recent flutter and dart sdk versions here: https://docs.flutter.dev/install/archive
+- check for breaking changes here: https://docs.flutter.dev/release/breaking-changes
+- adjust the `pubspec.yml` with the desired version `flutter` and `sdk`.
+- run in (/frontend):
+```
+fvm install <flutter-version>
+fvm use <flutter-version>
+```
+- adjust flutter and dart to the recent version in your IDE (f.e. Languages & Frameworks in IntelliJ).
+- run `fvm flutter pub get` to check if the packages run under the current dart and flutter environment.
+- build the app on android and ios and check if there are any issues.
```

Comment:
❓ I think it would be perfect to add the commands here so you don't have to search for them separately

The idea to add a doc is awesome 👍🏻

Reply:
We have all the runConfigs for that? I don't think you have to search for sth @seluianova

Reply:
ah, okay. I was thinking about this command... `bundle exec fastlane android build`